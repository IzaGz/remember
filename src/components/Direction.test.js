jest.mock('react-dom');
import React from 'react';

import renderer from 'react-test-renderer';
import Direction from './Direction';

test('Direction test', () => {
  const params = {
    geo: {
      longitude: 51.49948,
      latitude: -0.12481
    },
    longitude: 51.50136,
    latitude: -0.14189,
    dist: 2.12345
  };

  const component = renderer.create(<Direction {...params} />);
  expect(component.toJSON()).toMatchSnapshot();
});

test('Proximity test', () => {
  const params = {
    geo: {
      longitude: 1,
      latitude: 1
    },
    longitude: 1,
    latitude: 1,
    dist: 0.019
  };

  const component = renderer.create(<Direction {...params} />);
  expect(component.toJSON()).toMatchSnapshot();
});

test('Pre population test', () => {
  const component = renderer.create(<Direction/>);
  expect(component.toJSON()).toMatchSnapshot();
});
