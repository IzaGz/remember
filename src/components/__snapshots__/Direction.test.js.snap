exports[`test Direction test 1`] = `
<i
  className="material-icons"
  style={
    Object {
      "transform": "rotate(173deg)"
    }
  }>
  navigation
</i>
`;

exports[`test Pre population test 1`] = `<div />`;

exports[`test Proximity test 1`] = `
<i
  className="material-icons">
  star
</i>
`;
